Safety/Inc/PWM.hpp

```diff
@@ -22,9 +22,29 @@
 #pragma once
 
 #include <stdint.h>
+#include "stm32f0xx_hal.h"
+
+#define DSHOT_DATA_FRAME_LEN 16
+#define DSHOT_DMA_BUFFER_SIZE 18
+#define DSHOT_MAX_THROTTLE 2047
+#define DSHOT_BIT_1 240
+#define DSHOT_BIT_0 120
+#define NUM_DSHOT_MOTORS 4
```

Comment:
I have moved the #defines to the .cpp file, but is it convention to use constant variables rather than #define in cpp? I saw that MAX_CHANNELS and NUM_AVAILABLE_CHANNELS are #defines so I did the same?